core/mbed/uvisor-lib/box_config.h

```diff
@@ -48,35 +53,35 @@ UVISOR_EXTERN const uint32_t __uvisor_mode;
 /* this macro selects an overloaded macro (variable number of arguments) */
 #define __UVISOR_BOX_MACRO(_1, _2, _3, _4, NAME, ...) NAME
 
-#define __UVISOR_BOX_CONFIG(box_name, acl_list, acl_list_count, stack_size, context_size) \
```

Comment:
@Patater please keep `box_name` as it refers to the "variable name" of  the box. `uvisor_box_name_` on the other hand refers to the box prefix, so __uvisor_box_prefix makes more sense in this context.